app/lib/submission_builder/ty2024/states/md/documents/md502.rb

```diff
@@ -140,6 +140,23 @@ def document
         end
       end
       xml.TaxWithheld calculated_fields.fetch(:MD502_LINE_40)
+      add_element_if_present(xml, "AuthToDirectDepositInd", :MD502_AUTHORIZE_DIRECT_DEPOSIT)
+      if @intake.payment_or_deposit_type.to_sym == :direct_deposit
```

Comment:
i feel a bit confused that the joint account holder info has nothing to do with the payment/deposit type? is there a case where the type is _not_ direct deposit but there _is_ a joint account holder?

Reply:
oh you're right, it needs to be nested under this too